app/assets/stylesheets/components/_filter-panel.scss

```diff
@@ -15,6 +15,7 @@
   gap: govuk-spacing(2);
 
   .gem-c-heading {
```

Comment:
We shouldn't be styling anything with a `.gem-c-` prefix in applications. This breaks our model of component isolation. If changes are needed they should be made directly to the component in `govuk_publishing_components` - or applied to surrounding elements (the margin-left could easily be done as padding left on the wrapper for this heading component).

Reply:
in actual fact dont need the style at all, the two elements used to b the other way round, think thats why it was there